### ISSUE TITLE
Old School Sources

### ISSUE THREAD

**Issue by blakegearin** [blakegearin/github-custom-global-navigation#1]
This thread is for users to contribute sources for the [Old School configuration](https://github.com/blakegearin/github-custom-global-naviation#old-school). Preferably they show navigation around the GitHub website, especially less frequented pages.

**Comment by blakegearin** [comment]
I have a few updates to share.

- I haven't been able to find new video sources on Twitch, though this is not especially surprising since it seems a lot of creators don't use it for long-term video storage

- I was able to use Google's advanced search to look for YouTube videos in early 2023 and it works well: [query](https://www.google.com/search?q=site%3Ayoutube.com+how+to+github&sca_esv=579571626&source=lnt&tbs=cdr%3A1%2Ccd_min%3A1%2F1%2F2023%2Ccd_max%3A3%2F31%2F2023&tbm=)

- I found that the [Features](https://github.com/features) page has not been transitioned yet!
  - I went ahead and did a full backup by exporting my cookies (useful extension: [Get cookies.txt LOCALLY](https://chrome.google.com/webstore/detail/get-cookiestxt-locally/cclelndahbckbenkjhflpdbgdldlbecc)) then running `wget -r -E -k -p -np -nc --random-wait --load-cookies cookies.txt https://github.com/features/`

This is very exciting as getting the code directly from the source is essentially the holy grail.

I'll be doing some exploratory research into adding an option where the exact old code is dropped in as a full `<header>` replacement. I know there will be some complexity though in terms of search and the right sidebar since they're somewhat stateful. It's unclear what will be easier to maintain long-term, this old element that's mostly static or what I have now which makes updates to emulate the old look & feel. Either way, it's a nice to have options.

**Comment by dmitry-stepanenko** [comment]
Have you tried searching here https://web.archive.org/? E.g. https://web.archive.org/web/20230303211904/https://github.com/angular/angular

**Comment by blakegearin** [comment]
@dmitry-stepanenko Yes, but as far as I can tell the Wayback Machine only contains website snapshots that are public-facing instead of logged in, likely to not expose private data. The navigation is different in these two contexts, and I'm not too interested in modifying the public-facing version.

**Comment by eshack94** [comment]
Just curious — is this project still active?

**Comment by blakegearin** [comment]
I'm not actively working on this project at the moment. But I daily drive Old School and Happy Medium on different devices, so I plan on continuing to support this long-term.

For reference, I began working on integrating the raw HTML header I mentioned on Nov 5, 2023. I had to examine it all to find stateful & user-specific values, then add new logic to fill those in so it works as expected. So it wasn't as simple of an addition as I thought. I also found it a bit tricky to communicate the difference between this offering and the existing Old School configuration, since in theory they should be identical in appearance.
